crmsh/bootstrap.py

```diff
@@ -104,6 +105,179 @@ def init_qdevice(self):
                 cmds=self.qdevice_heuristics,
                 mode=self.qdevice_heuristics_mode)
 
+    def init_sbd_manager(self):
+        self.sbd_manager = SBDManager(self.sbd_device, self.diskless_sbd)
+
+
+class SBDManager(object):
+    """
+    Class SBDManager is for sbd configuration and sbd service management
+    """
+    sbd_config_dict = {
```

Comment:
This variable is not so properly used. You declare here as constant or class method, but you change it for each instance. That's not a good idea.  If you are using for a particular object instance class (what you do right now), just declare this method like the others, and not as class method

Reply:
@arbulu89 , I put `sbd_config_dict` into function `_update_configuration` as a internal variables, what do you think of this change?